content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md

```diff
@@ -0,0 +1,481 @@
+---
+title: 'Door Intruder Detector Using ML with the Nicla Voice'
+description: "This application note describes how to identify when intruders are trying to force an entrance door by analyzing surroundings sounds using a Nicla Voice, Machine Learning Tools, and the Arduino IoT Cloud"
+difficulty: intermediate
+tags:
+  - Intruder detector
+  - Nicla
+  - Portenta
+  - BLE
+  - Machine Learning
+  - Edge Impulse
```

Comment:
Please remove Edge Impulse and just mention Machine Learning Tools instead

Reply:
done